### DISCUSSION TITLE
【Bug】Untranscribed

### DISCUSSION THREAD

**Discussion by ioudove** [openai/whisper#1725] [Q&A]
I tried several times and could never get the transcription to work. whisper ran it and generated no subtitles whatsoever. I tried converting the original video to m4a, wma, mp3 none of which could be transcribed. [Click to view file](https://drive.google.com/file/d/1Dm9Is8_mO8vnpKJJINquIadgWGVLQvvO/view?usp=share_link)
My Operations (in Colab)：
!pip install openai-whisper
!whisper 210222晚上辅导.m4a --model large --language zh --output_format srt

**Comment by ioudove** [comment]
I've tried multiple times to transcribe this file, does anyone know what the problem is? And what should I do to fix it?

**Comment by misutoneko** [comment]
Hi! I gave it a go, your sample seems to work fine here...
I used the whisper_timestamped variant (and small model), but it shouldn't make much difference.
This sample is over five hours long, so it took a very long time to finish on my modest hw.

If this was your first try with whisper, I'd suggest trying with a smaller sample first (until you've verified that the process itself works).
Also, you didn't post any logs. Was there just nothing interesting there?

Btw the models assume 16kHz .wav input, but the conversion is handled on the fly I think (needs ffmpeg).
You can do the conversion manually too if you want, the command is something like:
`ffmpeg -y -i myfile.m4a -ar 16000 -ac 1 -c:a pcm_s16le myfile.wav`

**Reply by ioudove** [reply]
How are you?
1, Can you share your successfully transcribed srt file with me?
2, it's not my first time to use Whisper, I've tried this file with large, medium and small, but I failed all of them
3, I didn't post any logs because I don't really play with Github, I'm not familiar with it. And I don't know how to program.
4. 16kHz .wav I will try it.
Thank you very much for replying to me :D

**Reply by misutoneko** [reply]
Sure, here it is.
[whisper_1725_transcribed.tar.gz](https://github.com/openai/whisper/files/13069137/whisper_1725_transcribed.tar.gz)

So if I understood this correctly you have previously transcribed other sound samples successfully, just not this one?
Were they similar in size/duration?
Maybe you could try splitting the file into smaller pieces (with ffmpeg).
I've never used Colab, but one would assume that the program's output would be kinda essential feature to have :D
Those logs might tell you immediately what the problem is.

**Reply by ioudove** [reply]
Yes, I've been using Colab frequently to run Whisper recently.
The subtitle files you provided are excellent; the timing is mostly accurate.
